manifests/init.pp

```diff
@@ -67,38 +84,37 @@
   windowsfeature { "${_web_asp_net}":
   } ->
 
-  # remove default web site
-  iis::manage_site {'Default Web Site':
-    ensure    => absent,
-    site_path => 'any',
-    app_pool  => 'DefaultAppPool',
-    require   => Windowsfeature['Web-WebServer'],
```

Comment:
Also note that if you just `include chocolatey_server` it will disable instead of delete the default website. 👍